testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -175,6 +177,18 @@ QUnit.test("item content should be rendered correctly when template was changed
     assert.equal($items.eq(1).text(), "Test2", "element has correct content");
 });
 
+QUnit.test("Add touch action style for IE and Edge", function(assert) {
+    if(!browser.msie) {
```

Comment:
You can rid of using `return`. Just define a variable like that

    var touchActionClassShouldApplied = browser.msie;

And after that change the assertion like below

    assert.equal(wrapperHasTouchActionClass, touchActionClassShouldApplied)